config/environments/development.rb

```diff
@@ -72,5 +72,5 @@
   config.sprint_uri = 'http://example_sprint.com/graphql'
 
   # FreshService URL
-  config.fresh_sevice_new_ticket_url = 'https://sanger.freshservice.com/a/tickets/new'
+  config.fresh_sevice_new_ticket_url = 'https://example.freshservice.com'
 end
```

Comment:
are you sure this is the right URL  that should be used, 
I thought we need to point to one of our items, but also I can not find gatekeeper, but this is should be included within SS ?